src/tests.builds

```diff
@@ -13,7 +13,10 @@
   <Import Project="$(ToolsDir)VersionTools.targets" Condition="Exists('$(ToolsDir)VersionTools.targets')" />
 
   <ItemGroup>
-    <Project Include="$(MSBuildThisFileDirectory)*\tests\**\*.Tests.csproj" Exclude="@(TestProjectExclusions)">
+    <Project Condition="'$(Performance)'=='true'" Include="$(MSBuildThisFileDirectory)System.Collections*\tests\**\*Performance.Tests.csproj" Exclude="@(TestProjectExclusions)">
+      <AdditionalMetadata>ConfigurationGroup=$(ConfigurationGroup)</AdditionalMetadata>
+    </Project>
+	<Project Condition="'$(Performance)'!='true'" Include="$(MSBuildThisFileDirectory)*\tests\**\*.Tests.csproj" Exclude="@(TestProjectExclusions)">
```

Comment:
Yes.  I wanted to limit to just the performance projects while running performance because once a test run is finished we expect certain output, and that would not be produced by a non-performance test project.  Rather then doing the checking in generated RunTests.cmd I just filter to only performance tests.